proto/redpanda/api/dataplane/v1alpha2/cloud_storage.proto

```diff
@@ -0,0 +1,393 @@
+    STATE_PREPARED = 3;
+    // Executing: The mount task is actively transferring or transforming data.
+    STATE_EXECUTING = 4;
+    // Executed: The core mount task actions are complete, but the mount task has not yet cut over or finalized.
+    STATE_EXECUTED = 5;
+    // Cut Over: The mount task has reached a critical point where ownership is transferred or final adjustments are made.
+    STATE_CUT_OVER = 6;
+    // Finished: The mount task has been successfully completed, and no further actions are required.
+    STATE_FINISHED = 7;
+    // Canceling: The mount task is in the process of being canceled, and rollback or cleanup actions may be in progress.
+    STATE_CANCELING = 8;
+    // Cancelled: The mount task has been fully canceled, and no further actions will be taken.
+    STATE_CANCELLED = 9;
+  }
+
+  // Id is the unique identifier for this mount task.
+  int32 id = 1;
+  // State describes the current state of the mount task (e.g. "cancelled").
+  State state = 2;
+  // Type describes the type of this task (mount or unmount).
+  Type type = 3;
+  // Topics is the list of topics that are being mounted or unmounted.
+  repeated Topic topics = 4;
+}
+
+message ListMountTasksRequest {}
+
+message ListMountTasksResponse {
+  repeated MountTask tasks = 1;
+}
+
+message GetMountTaskRequest {
+  // ID is the unique identifier of the mount task to retrieve.
+  int32 id = 1 [
+    // We won't add validation required here, because 0 may be a valid id.
+    (google.api.field_behavior) = REQUIRED
+  ];
+}
+
+message GetMountTaskResponse {
+  MountTask task = 1;
+}
+
+message DeleteMountTaskRequest {
+  // ID is the unique identifier of the mount task to delete.
+  int32 id = 1 [(google.api.field_behavior) = REQUIRED /* We won't add validation required here, because 0 may be a valid id. */];
+}
+
+message DeleteMountTaskResponse {}
+
+message UpdateMountTaskRequest {
+  enum Action {
+    ACTION_UNSPECIFIED = 0;
+    ACTION_PREPARE = 1;
+    ACTION_EXECUTE = 2;
+    ACTION_FINISH = 3;
+    ACTION_CANCEL = 4;
+  }
+
+  // ID is the unique identifier of the mount task to delete.
+  int32 id = 1 [(google.api.field_behavior) = REQUIRED /* We won't add validation required here, because 0 may be a valid id. */];
+
+  // Action is the action to execute on mount task.
+  Action action = 2 [
+    (google.api.field_behavior) = REQUIRED,
+    (buf.validate.field).required = true,
+    (buf.validate.field).enum = {
+      defined_only: true
+      not_in: [0] /* Disallow: Unspecified */
+    }
+  ];
+}
+
+message UpdateMountTaskResponse {}
+
+service CloudStorageService {
+  rpc MountTopics(MountTopicsRequest) returns (MountTopicsResponse) {
+    option (google.api.http) = {
+      post: "/v1alpha2/cloud-storage/topics/mount"
+      body: "topics"
+      response_body: "*"
+    };
+    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      summary: "Mount topics from tiered storage"
```

Comment:
```suggestion
      summary: "Mount topics from object storage"
```
I think we could be even more specific and say that it's the topics that have been specifically unmounted to the cloud (and not just "topics that have Tiered Storage enabled")

Reply:
I find "from tiered storage" more descriptive for those that do not know what mounting/unmounting means. We may have users scrolling through the documentation who are not familiar (nor using) Redpanda. Happy to include both though?

Reply:
@weeco sure, something like "Mount topics to a cluster from object storage"? (or Tiered Storage, just as long as we can be consistent in what we use in the summary since they will show as subheadings and nav elements too in the reference doc)